Resources/Prototypes/Recipes/Construction/structures.yml

```diff
@@ -145,6 +145,78 @@
   placementMode: SnapgridCenter
   canRotate: false
 
+- type: construction
+  name: directional window
+  id: WindowDirectional
+  graph: windowDirectional
+  startNode: start
+  targetNode: windowDirectional
+  category: Structures
+  description: Clear.
+  canBuildInImpassable: true
+  conditions:
+    - !type:EmptyOrWindowValidInTile {}
+    - !type:NoWindowsInTile {}
```

Comment:
should this be a condition? i think you should be able to construct multiple directional windows in a tile

Reply:
I tried removing nowindowsintile, but it doesn't let you build 2 in the same tile anyways. Doesn't really matter; i don't think any spots in either map have 2 in the same tile anyways, and it's rare it'll actually affect gameplay.

Reply:
ehh, should be fine then